ODIN_II/regression_test/tools/run_yosys.sh

```diff
@@ -362,7 +362,7 @@ function run_single_hdl() {
     for circuit in "${_VERILOG_INPUT_LIST[@]}"
     do
         # validate input file
-        if [ "${circuit: -2}" != ".v" ]; then
+        if ["${circuit: -2}" != ".sv" and "${circuit: -2}" != ".v" ]; then
```

Comment:
this should be `"${circuit: -3}" != ".sv"`

Reply:
there is also a script in `$VTR_ROOT/ODIN_II/regression_test/tools/synth.tcl` that needs to be modified